tools/admin/wskadmin

```diff
@@ -132,6 +132,8 @@ def parseArgs():
     subcmd.add_argument('--firesPerMinute', help='trigger fires per minute allowed', type=int)
     subcmd.add_argument('--concurrentInvocations', help='concurrent invocations allowed for this namespace', type=int)
     subcmd.add_argument('--allowedKinds', help='list of runtime kinds allowed in this namespace', nargs='+', type=str)
+    subcmd.add_argument('--enableStoreActivations', help='enable storing of activations to datastore for this namespace', default=None, dest='storeActivations', action='store_true')
+    subcmd.add_argument('--disableStoreActivations', help='disable storing of activations to datastore for this namespace', default=None, dest='storeActivations', action='store_false')
```

Comment:
Thinking about scriptability here: Would it be easier to just use a value `storeActivations=true/false` instead of two distinct flags? If I were to use `wskadmin` in a script, I much rather do `wskadmin --storeActivations=$VALUE` than replacing the whole flag if that makes sense?

Reply:
I can have another look on that one 👍